osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-external-interface.xsd

```diff
@@ -28,7 +28,7 @@
           <xsd:element ref="bun:GetPeriodicMeterReadsRequest"/>
           <xsd:element ref="bun:GetPowerQualityProfileRequest"/>
           <xsd:element ref="adm:SetMaintenanceStateRequest"/>
-          <xsd:element ref="bun:FindEventsRequest"/>
```

Comment:
My remarks about events versus outages aside, it seems illogical to replace `FindEventsRequest` by `GetOutagesRequest` while adding the `GetOutagesResponse` as an alternative to the `FindEventsResponse`.
Is this a mistake or am I missing something where these two requests and responses don't have to match in pairs?